File: Python/selection_sort/selection_sort.py
def selection_sort(arr):
    for i in range(len(arr)):
        min_id = i;
        for j in range(i+1,len(arr)):
            if arr[min_id] > arr[j]:
                min_id = j;
        temp = arr[min_id];
        arr[min_id] = arr[i];
        arr[i] = temp;
    return arr;

print(selection_sort([1,23,13,514,123,34,1,231,2]))

File: JavaScript/selection_sort/selection_sort.js
function selection_sort(arr) {
 for (let i = 0; i < arr.length ; i++) {
   let min_id = i;
   for (let j  = i + 1; j  < arr.length; j ++) {
     if (arr[min_id] > arr[j]) {
       min_id = j;
     }
   }
   let temp = arr[min_id];
   arr[min_id] = arr[i];
   arr[i] = temp;
 } 
  return arr;
}

console.log(selection_sort([1,23,1,51,23,12,32,123,3]))
